src/axes/numericAxis.ts

```diff
@@ -342,14 +341,8 @@ export class Numeric extends Axis<number> {
     for (let i = 0; i < rects.length - (interval); i += interval) {
       let currRect = rects[i];
       let nextRect = rects[i + interval];
-      if (this.isHorizontal()) {
-        if (currRect.right + padding >= nextRect.left) {
-          return false;
-        }
-      } else {
-        if (currRect.top - padding <= nextRect.bottom) {
-          return false;
-        }
+      if (Utils.DOM.clientRectsOverlap(currRect, nextRect)) {
```

Comment:
im not sure this matches the same logic, especially it doesn't consider the padding.